src/frontends/pytorch/src/op/cat.cpp

```diff
@@ -197,6 +197,18 @@ OutputVector translate_vstack(const NodeContext& context) {
     return out;
 };
 
+OutputVector translate_quantized_relu6(const NodeContext& context) {
```

Comment:
we have already implementation for it. See openvino\src\frontends\pytorch\src\op\relu6.cpp